src/flask_track_usage/storage/sql.py

```diff
@@ -121,6 +121,10 @@ def store(self, data):
         """
         user_agent = data["user_agent"]
         utcdatetime = datetime.datetime.fromtimestamp(data['date'])
+        if data["ip_info"]:
+            ip_info_str = json.dumps(data["ip_info"])[:128]
```

Comment:
This could lead to loadable json later on. What about something like:

```python
for x in range(len(data["ip_info"].keys())):
    if len(json.dumps(data["ip_info"])) <= 10:
        break
    t.pop(list(data["ip_info"].keys())[0])

ip_info_str = json.dumps(t)
```